app/uk/gov/hmrc/hec/repos/HECTaxCheckStore.scala

```diff
@@ -119,7 +119,7 @@ class HECTaxCheckStoreImpl @Inject() (
           }
 
       override def ensureIndexes(implicit ec: ExecutionContext): Future[Seq[Boolean]] = {
-        super.ensureIndexes
+        val _ = super.ensureIndexes
```

Comment:
this is gonna change soon anyways, but I think we probably should have mapped into `super.ensureIndexes` to make sure that was done before the `Future` below it ran instead of both the things happening on separate threads

Reply:
updated